### PR TITLE
mattly's mirrored centromere layout

### PR DESCRIPTION
**Pull request by mattly**
## Description

I added a layout for my centromere, and I believe this is the first user layout for this keyboard.

Since there wasn't a build guide, I soldered the switches in upside-down, so the layout is mirrored. If someone knows a better way to handle swapping the keycodes, I'd appreciate it.

For some reason it's also powering-on into the second layer (`OVER_WIN`) instead of the first layer. It's annoying but if anyone knows why I'd appreciate it.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [ ] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [ ] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [ ] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/centromere/keymaps/mattly/keymap.c [hunk 1]
```diff
@@ -0,0 +1,56 @@
+#include QMK_KEYBOARD_H
+#include "mattly.h"
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+  // I apparently soldered in my switches on the wrong sides of the boards, so this is mirrored
+
+  [_BASE_MAC] = LAYOUT_split_3x6_3( \
```
Comment (fauxpark) [inline keyboards/centromere/keymaps/mattly/keymap.c:8]:
```suggestion
  [_BASE_MAC] = LAYOUT_split_3x6_3(
```

File: keyboards/centromere/keymaps/mattly/rules.mk [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = caterina
+
+MOUSEKEY_ENABLE = no	# Mouse keys
```
Comment (fauxpark) [inline keyboards/centromere/keymaps/mattly/rules.mk:14]:
```suggestion
MOUSEKEY_ENABLE = no
```

**Review by fauxpark — CHANGES_REQUESTED** [review]

File: keyboards/centromere/keymaps/mattly/keymap.c (hunk 1 above)
Comment (mattly) [reply keyboards/centromere/keymaps/mattly/keymap.c:8]: ah, this is leftover from the `default` keymap which I had copied.

**Review by fauxpark — APPROVED** [review]

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**